crates/thetawave_interface/src/options/mod.rs

```diff
@@ -0,0 +1,52 @@
+use std::{
```

Comment:
Now that I look at this again, I dont like this in the interface crate. How this stuff is parsed + CLI is detinitely implementation. Let's move this back to the main `src/` crate.